utils/data.py

```
@@ -1,104 +1,89 @@
-    data = load_json(filename)
-
+    data = await load_json(filename)
     # ID 자동 생성
     if id_field not in item:
         item[id_field] = get_next_id(data, id_field)
-
     # 생성 시간 추가
     if "createdAt" not in item:
-        item["createdAt"] = datetime.now(timezone.utc).isoformat()
-
+        item["createdAt"] = datetime.now().isoformat()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)